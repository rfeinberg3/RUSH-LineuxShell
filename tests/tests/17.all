rush> rush> zebra
rush> rush> An error has occurred
rush> An error has occurred
rush> 