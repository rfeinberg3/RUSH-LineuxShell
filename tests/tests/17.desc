Check implementation of built-in commands (path).
