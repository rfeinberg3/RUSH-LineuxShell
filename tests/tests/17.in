path /autograder/source/bin
p1.sh
path
p1.sh
ls
exit
