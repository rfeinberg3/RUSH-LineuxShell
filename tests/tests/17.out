rush> rush> zebra
rush> rush> rush> rush> 